### PR TITLE
Add status to cli

### PR DESCRIPTION
**Pull request by luisremis**
```
remis@server:~/ad/aperturedb$ adb config create yfcc
Enter aperturedb host name [localhost]: yfcc100m.datasets.gcp.aperturedata.io
Enter aperturedb port number [55555]:
Enter aperturedb username [admin]:
Enter aperturedb password [admin]:
Use REST [y/N]: N
Use SSL [Y/n]: Y

remis@server:~/ad/aperturedb$ adb config activate yfcc --as-global

remis@server:~/ad/aperturedb$ adb utils execute status
[
    {
        "GetStatus": {
            "info": "OK",
            "status": 0,
            "system": "ApertureDB",
            "version": "0.16.4"
        }
    }
]
```

### CODE REVIEW

File: aperturedb/cli/utilities.py [hunk 1]
```diff
@@ -24,10 +25,15 @@ def confirm(command: CommandTypes):
     return True
 
 
+def print_utils(utils):
+    print(utils)
+
+
```
Comment (gsaluja9) [inline aperturedb/cli/utilities.py:31]:
```suggestion
```

File: aperturedb/cli/utilities.py [hunk 2]
```diff
@@ -24,10 +25,15 @@ def confirm(command: CommandTypes):
...
 @app.command()
 def execute(command: CommandTypes):
     utils = Utils(create_connector())
     available_commands = {
+        CommandTypes.STATUS: lambda: print_utils(utils),
```
Comment (gsaluja9) [inline aperturedb/cli/utilities.py:36]:
```suggestion
        CommandTypes.STATUS: lambda: print(utils),
```

**Review by gsaluja9 — COMMENTED** [review]
minor suggestions to make it a cleaner read.

**Review by gsaluja9 — APPROVED** [review]

File: aperturedb/cli/utilities.py (hunk 2 above)
Comment (luisremis) [reply aperturedb/cli/utilities.py:36]: oh, nice. I tried exactly this first, but I was getting and error. That's why I had to add that extra function (I didn't like it either). Would you try it and tell me if it works for you?

**Review by gsaluja9 — APPROVED** [review]
yes this works.

**Pull request merged**